### DISCUSSION TITLE
Metadata columns from Cellragner arc

### DISCUSSION THREAD

**Discussion by Chhiring-Lama** [stuart-lab/signac#836] [Q&A] [answered]
Hi, I am using the per_barcode_metrics.csv from cellranger-arc count as the metadata. I want to check percentage reads in DNase and blacklist ratio for QC. However, I am not able to find an equivalent to `DNase_sensitive_region_fragments` and `blacklist_region_fragments` in the metadata. Which columns can I use for the calculation?

**Comment by timoast** [comment, marked as answer]
Please see https://github.com/timoast/signac/discussions/709
